resources/views/components/dropdown-filter.blade.php

```diff
@@ -0,0 +1,16 @@
+<div class="filter p-0" >
+    <label for="{{ $id }}" class="bi bi-filter"> {{ $name }}
```

Comment:
label and select tags should be separated inside the div. select tag cannot be wrapped by label tag

Reply:
Yes got it...i refactored please review again